x11-misc/yabar/yabar-0.4.0_p20190316.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="a0d3fdfed992149b741eb8fcf53f02b5d1a6142e"
+MY_P="yabar-${MY_PV}"
+
+DESCRIPTION="A modern and lightweight status bar for X window managers."
+HOMEPAGE="https://github.com/geommer/yabar"
+SRC_URI="https://github.com/geommer/yabar/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="playerctl"
+
+DEPEND="
+	x11-libs/cairo[xcb]
+	x11-libs/pango
+	x11-libs/xcb-util-wm
+	x11-libs/gdk-pixbuf
+	x11-libs/libxkbcommon[X]
+	media-libs/alsa-lib
+	net-wireless/wireless-tools
+	dev-libs/libconfig
+	app-text/asciidoc
+	playerctl? ( media-sound/playerctl )
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+	# Declare VERSION without relying on git,
+	# and replace playerctl dependency option to base name
+	sed -i -e "s/\$(shell git describe)/0.4.0-179-ga0d3fdf/" \
+	-e "s/^DEPS += playerctl-1.0/DEPS += playerctl/" \
+	Makefile || die "Failed to update Makefile"
+}
+
+src_compile() {
+	if use playerctl; then
+		emake PLAYERCTL=1
+	else
+		emake
+	fi
+}
+
+src_install() {
+	default
+	docinto examples
+	dodoc "${S}"/examples/example.config
```

Comment:
Just a minor thing, but `${S}` should be the CWD.

Reply:
Fixed in newest push